package.json

```diff
@@ -114,7 +114,9 @@
     "moment": "^2.13.0",
     "node-fetch": "^1.6.0",
     "node-forge": "^0.7.1",
+    "node-watch": "^0.5.5",
     "opn": "^5.0.0",
+    "precinct": "^3.8.0",
```

Comment:
Could you also please add `package-lock.json` file?

Reply:
@RafalWilinski this has now been added